src/eynollah/eynollah.py

```diff
@@ -4487,12 +4487,9 @@ def run(self):
                     continue
                 else:
                     return pcgts
-
-            img_res, is_image_enhanced, num_col_classifier, num_column_is_classified = self.run_enhancement(self.light_version)
-            self.logger.info("Enhancing took %.1fs ", time.time() - t0)
-            #print("text region early -1 in %.1fs", time.time() - t0)
-            t1 = time.time()
```

Comment:
IMO it does not make sense to move that from the lower indentation level to all the conditional branches in a copycat fashion. `img_res` seems to be needed everywhere below, so why not just compute it once here?

(See follow-up comments below.)

Reply:
I have moved it up again, so that code does not need to be repeated.